src/search/ui/place.directive.html

```diff
@@ -21,8 +21,21 @@
     <div class="udb-place-city" ng-bind="event.address.addressLocality"></div>
   </div>
 
-  <div class="col-sm-2">
-    <span>permanent</span>
+  <div class="col-sm-2" ng-switch="event.calendarType">
+    <span ng-switch-when="permanent">permanent</span>
+    <span ng-switch-when="single">
+        <span class="udb-start-date" ng-bind="event.startDate | date: 'dd/MM/yyyy'"></span>
+    </span>
+    <span ng-switch-when="periodic" class="udb-date-range">
```

Comment:
Btw, everywhere in `place.directive.html` there are references to `event. ...` instead of `place. ...`. But this problem is rooted deep in the code so I wouldn't change it imo.